.github/workflows/test.yaml

```diff
@@ -7,13 +7,13 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-20.04, windows-2019]
-        python: [3.8. 3.9, 3.10, 3.11, 3.12]
```

Comment:
Missing `,` between 3.8 and 3.9 was leading to errors.

Reply:
After the fix - https://github.com/awslabs/aws-cloudformation-iam-policy-validator/actions/runs/8511270880